test/e2e/scenarios/ciliumeventobserver/scenario.go

```diff
@@ -0,0 +1,192 @@
+package ciliumeventobserver
+
+import (
+	"time"
+
+	"github.com/microsoft/retina/test/e2e/framework/kubernetes"
+	"github.com/microsoft/retina/test/e2e/framework/types"
+)
+
+const (
+	sleepDelay = 5 * time.Second
+	TCP        = "TCP"
+	UDP        = "UDP"
+
+	PolicyDenied = "POLICY_DENIED"
+)
+
+func ValidateCiliumEventObserverDropMetric() *types.Scenario {
+	name := "CiliumEventObserverDropMetric"
+	steps := []*types.StepWrapper{
+		{
+			Step: &kubernetes.CreateDenyAllNetworkPolicy{
+				NetworkPolicyNamespace: "kube-system",
+				DenyAllLabelSelector:   "app=agnhost-a",
+			},
+		},
+		{
+			Step: &kubernetes.CreateAgnhostStatefulSet{
+				AgnhostName:      "agnhost-a",
+				AgnhostNamespace: "kube-system",
+			},
+		},
+		{
+			Step: &kubernetes.ExecInPod{
+				PodName:      "agnhost-a-0",
+				PodNamespace: "kube-system",
+				Command:      "curl -s -m 5 bing.com",
+			},
+			Opts: &types.StepOptions{
+				ExpectError:               true,
+				SkipSavingParamatersToJob: true,
+			},
+		},
+		{
+			Step: &types.Sleep{
+				Duration: sleepDelay,
+			},
+		},
+		{
+			Step: &kubernetes.ExecInPod{
+				PodName:      "agnhost-a-0",
+				PodNamespace: "kube-system",
+				Command:      "curl -s -m 5 bing.com",
+			},
+			Opts: &types.StepOptions{
+				ExpectError:               true,
+				SkipSavingParamatersToJob: true,
+			},
+		},
+		{
+			Step: &kubernetes.PortForward{
+				Namespace:             "kube-system",
+				LabelSelector:         "k8s-app=retina",
+				LocalPort:             "9965",
```

Comment:
9965 is hubble port, can you set it as a const for reusability accross this file or e2e